chart/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "tesoro.fullname" . }}
+  labels:
+    {{- include "tesoro.labels" . | nindent 4 }}
+spec:
+  ports:
+    - port: 443
```

Comment:
Might be good to optionally expose the metrics port here too? In case people use servicemonitors in their setup?

Reply:
Super idea. When I created the chart I just copied the sample YAML:

* https://github.com/kapicorp/tesoro/blob/master/k8s/tesoro_service.yaml